seshdash/views.py

```diff
@@ -713,10 +712,11 @@ def get_latest_bom_data(request):
 def search(request):
     data=[]
     sites = Sesh_Site.objects.all()
```

Comment:
This is a bug. You should only be searching sites the user has permission to access.